src/Session.js

```diff
@@ -141,9 +141,12 @@ Session.prototype.logon = function(data) {
         send: function(method, data) { self.send(method, data); }
     }
 
-        log.info('User: ' + this.id + ' signed on');
-        this.currentRoom = this._server.getRoom(data.roomId);
-        setTimeout(function(){ self.subscribe(data); }, 500);
+    log.info('User: ' + this.id + ' signed on');
+    this.currentRoom = this._server.getRoom(data.roomId);
+    setTimeout(function(){ 
+        if (!self._socket.destroyed)
```

Comment:
This is more of a temp fix. Really, the mysql-auth plugin should take a callback so we can be sure it's done.